{"name": "Visualizar portal sucursal virtual -- @1.2 ", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.NoSuchWindowException: Message: no such window: target window already closed\nfrom unknown error: web view not found\n  (Session info: chrome=105.0.5195.102)\nStacktrace:\nBacktrace:\n\tOrdinal0 [0x00D7DF13+2219795]\n\tOrdinal0 [0x00D12841+1779777]\n\tOrdinal0 [0x00C2423D+803389]\n\tOrdinal0 [0x00C0C70A+706314]\n\tOrdinal0 [0x00C71502+1119490]\n\tOrdinal0 [0x00C74F7E+1134462]\n\tOrdinal0 [0x00C6E7C3+1107907]\n\tOrdinal0 [0x00C47F89+950153]\n\tOrdinal0 [0x00C48F56+954198]\n\tGetHandleVerifier [0x01072CB2+3040210]\n\tGetHandleVerifier [0x01062BB4+2974420]\n\tGetHandleVerifier [0x00E16A0A+565546]\n\tGetHandleVerifier [0x00E15680+560544]\n\tOrdinal0 [0x00D19A5C+1808988]\n\tOrdinal0 [0x00D1E3A8+1827752]\n\tOrdinal0 [0x00D1E495+1827989]\n\tOrdinal0 [0x00D280A4+1867940]\n\tBaseThreadInitThunk [0x7512FA29+25]\n\tRtlGetAppContainerNamedObjectPath [0x76F67A9E+286]\n\tRtlGetAppContainerNamedObjectPath [0x76F67A6E+238]\n\n", "trace": "  File \"C:\\Users\\gerso\\AppData\\Local\\Programs\\Python\\Python38\\lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"C:\\Users\\gerso\\AppData\\Local\\Programs\\Python\\Python38\\lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\loginsteps.py\", line 51, in closeBrwoser\n    context.driver.close()\n  File \"C:\\Users\\gerso\\AppData\\Local\\Programs\\Python\\Python38\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 550, in close\n    self.execute(Command.CLOSE)\n  File \"C:\\Users\\gerso\\AppData\\Local\\Programs\\Python\\Python38\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 428, in execute\n    self.error_handler.check_response(response)\n  File \"C:\\Users\\gerso\\AppData\\Local\\Programs\\Python\\Python38\\lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 243, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "steps": [{"name": "Given launch chrome browser", "status": "passed", "start": 1662742713105, "stop": 1662742714523}, {"name": "When carga portal sucursal virtual", "status": "passed", "start": 1662742714524, "stop": 1662742717644}, {"name": "And Ingresar rut 14044384-0", "status": "passed", "start": 1662742717644, "stop": 1662742717775}, {"name": "And Ingresar password 111111", "status": "passed", "start": 1662742717775, "stop": 1662742717866}, {"name": "And Click en boton ingresar", "status": "passed", "start": 1662742717866, "stop": 1662742717953}, {"name": "Then valida ingreso", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.NoSuchWindowException: Message: no such window: target window already closed\nfrom unknown error: web view not found\n  (Session info: chrome=105.0.5195.102)\nStacktrace:\nBacktrace:\n\tOrdinal0 [0x00D7DF13+2219795]\n\tOrdinal0 [0x00D12841+1779777]\n\tOrdinal0 [0x00C2423D+803389]\n\tOrdinal0 [0x00C0C70A+706314]\n\tOrdinal0 [0x00C71502+1119490]\n\tOrdinal0 [0x00C74F7E+1134462]\n\tOrdinal0 [0x00C6E7C3+1107907]\n\tOrdinal0 [0x00C47F89+950153]\n\tOrdinal0 [0x00C48F56+954198]\n\tGetHandleVerifier [0x01072CB2+3040210]\n\tGetHandleVerifier [0x01062BB4+2974420]\n\tGetHandleVerifier [0x00E16A0A+565546]\n\tGetHandleVerifier [0x00E15680+560544]\n\tOrdinal0 [0x00D19A5C+1808988]\n\tOrdinal0 [0x00D1E3A8+1827752]\n\tOrdinal0 [0x00D1E495+1827989]\n\tOrdinal0 [0x00D280A4+1867940]\n\tBaseThreadInitThunk [0x7512FA29+25]\n\tRtlGetAppContainerNamedObjectPath [0x76F67A9E+286]\n\tRtlGetAppContainerNamedObjectPath [0x76F67A6E+238]\n\n", "trace": "  File \"C:\\Users\\gerso\\AppData\\Local\\Programs\\Python\\Python38\\lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"C:\\Users\\gerso\\AppData\\Local\\Programs\\Python\\Python38\\lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\loginsteps.py\", line 51, in closeBrwoser\n    context.driver.close()\n  File \"C:\\Users\\gerso\\AppData\\Local\\Programs\\Python\\Python38\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 550, in close\n    self.execute(Command.CLOSE)\n  File \"C:\\Users\\gerso\\AppData\\Local\\Programs\\Python\\Python38\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 428, in execute\n    self.error_handler.check_response(response)\n  File \"C:\\Users\\gerso\\AppData\\Local\\Programs\\Python\\Python38\\lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 243, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "start": 1662742717953, "stop": 1662742725316}], "parameters": [{"name": "rut", "value": "14044384-0"}, {"name": "password", "value": "111111"}], "start": 1662742713104, "stop": 1662742725317, "uuid": "3f1002ce-98df-4bbb-9d34-4acb5f597528", "historyId": "4dd743e6b3f9d4255ef3ee0e5eee8b55", "fullName": "Login en nuevaweb: Visualizar portal sucursal virtual", "labels": [{"name": "severity", "value": "normal"}, {"name": "feature", "value": "Login en nuevaweb"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}]}